clang/lib/Sema/SemaType.cpp

```diff
@@ -2312,6 +2312,18 @@ QualType Sema::BuildArrayType(QualType T, ArraySizeModifier ASM,
   return T;
 }
 
+bool CheckBitIntElementType(Sema &S, SourceLocation AttrLoc,
+                            const BitIntType *BIT, bool ForMatrixType = false) {
+  // Only support _BitInt elements with byte-sized power of 2 NumBits.
+  unsigned NumBits = BIT->getNumBits();
+  if (!llvm::isPowerOf2_32(NumBits) || NumBits < 8) {
+    S.Diag(AttrLoc, diag::err_attribute_invalid_bitint_vector_type)
+        << ForMatrixType << (NumBits < 8);
+    return true;
```

Comment:
AFAIK you can do
```suggestion
    return S.Diag(AttrLoc, diag::err_attribute_invalid_bitint_vector_type)
        << ForMatrixType << (NumBits < 8);
```

Reply:
Ah yeah, I always forget that you can do this.